vonk/facade/databasemodel.rst

```diff
@@ -80,15 +74,29 @@ Clean up generated code
 Create your first mapping
 -------------------------
 
-#. Add a project folder Repository.
-#. Add a new class ``ResourceMapper``
-#. Add usings for ``Vonk.Core.Common`` and ``Vonk.Facade.Starter.Models``
+#. Add a new class ``ResourceMapper`` to the project
+#. Add usings for ``Vonk.Core.Common``, for ``Hl7.Fhir.Model`` and for ``<your project>.Models``
 #. Add a method to the class ``public IResource MapPatient(ViSiPatient source)``
-#. In this method, put code to create a FHIR Patient object, and fill it's elements with data from the ViSiPatient.
-#. Then return the created Patient object as an IResource (you can use the extension method ``AsIResource``).
+#. In this method, put code to create a FHIR Patient object, and fill its elements with data from the ViSiPatient:
+
+   .. code-block:: c#
+
+     var patient = new Patient
+     {
+         Id = source.Id.ToString(),
+         BirthDate = new FhirDateTime(new DateTimeOffset(source.DateOfBirth)).ToString()
```

Comment:
Changed that, but needs to change in the project code as well

Reply:
Project code only sets the date, with source.DateOfBirth.ToFhirDate(). This is actually the correct way, since the birthdate field is of type date and not dateTime.
Need to fix this in the docs, not in the code.

Reply:
done